src/pkg/manifests/package:pkg.p5m

```diff
@@ -497,6 +498,6 @@ depend type=require fmri=library/python/prettytable-35
 depend type=require fmri=library/python/pybonjour-35
 depend type=require fmri=library/python/pycurl-35
 depend type=require fmri=library/python/pyopenssl-35
-depend type=require fmri=library/python/simplejson-35@3.6.5
+depend type=require fmri=library/python/rapidjson-35@3.6.5
```

Comment:
rapidjson is at 0.9.1, please, remove version requirement

Reply:
@citrus-it

Reply:
Sorry, done!